app/controllers/api/integrations_controller.rb

```diff
@@ -4,6 +4,7 @@ class Api::IntegrationsController < Api::BaseController
   before_action :find_service
   before_action :find_proxy
   before_action :authorize
+  before_action :unauthorize_apiap, only: %i[edit update]
```

Comment:
`unauthorize` sounds pretty weird to me.. what about simple `raise_if_apiap` ?

Reply:
@duduribeiro @macejmic WDYT about `raise_not_found_if_apipap`? because simply raise... it is like 'raise what?'